cinema/models.py

```diff
@@ -75,9 +75,7 @@ def __str__(self):
 
 class Order(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
-    user = models.ForeignKey(
-        settings.AUTH_USER_MODEL, on_delete=models.CASCADE
-    )
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
```

Comment:
Ensure that the `AUTH_USER_MODEL` is correctly configured in your Django settings. This is crucial for the foreign key relationship with the `Order` model.